diff --git a/kernel/sucompat.c b/kernel/sucompat.c
index 3ef005fc..d19f18fa 100644
--- a/kernel/sucompat.c
+++ b/kernel/sucompat.c
@@ -368,7 +368,7 @@ void ksu_sucompat_exit()
 #ifdef CONFIG_KSU_SUSFS_SUS_SU
 extern bool ksu_su_compat_enabled;
 bool ksu_devpts_hook = false;
-bool susfs_is_sus_su_hooks_enabled __read_mostly = false;
+extern bool susfs_is_sus_su_hooks_enabled;
 int susfs_sus_su_working_mode = 0;
 
 static bool ksu_is_su_kps_enabled(void) {
